addons/event/models/event_event.py

```diff
@@ -170,12 +170,13 @@ def _get_default_stage_id(self):
     date_end = fields.Datetime(string='End Date', required=True, tracking=True)
```

Comment:
Also, in this commit: "[IMP] event: adds graph view and adds change for event_barcode"

You should include a link to the enterprise PR for reference:

odoo/enterprise#7760

(Not 100% sure about syntax, check if github understands that).